src/AppBundle/Utils/ListUtils/ListModels/ListModelsObtentionDistinctions.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+
+namespace AppBundle\Utils\ListUtils\ListModels;
+
+use AppBundle\Entity\Membre;
+use AppBundle\Entity\ObtentionDistinction;
+use AppBundle\Utils\Event\EventPostAction;
+use AppBundle\Utils\ListUtils\AbstractList;
+use AppBundle\Utils\ListUtils\ActionLine;
+use AppBundle\Utils\ListUtils\ActionList;
+use AppBundle\Utils\ListUtils\Column;
+use AppBundle\Utils\ListUtils\ListModel;
+use AppBundle\Utils\ListUtils\ListRenderer;
+use Symfony\Component\Routing\Router;
+
+class ListModelsObtentionDistinctions extends AbstractList
+{
+    public function getDefault($items, $url = null)
+    {
+        $twig = $this->twig;
```

Comment:
Quel est le but de getDefault ? pas moyen de le remplacer ? et $twig = $this->twig :D